hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,20 @@
+RELEASE_TYPE: patch
```

Comment:
opinions welcom as to if this should be a patch or no

Reply:
My rule of thumb is "if in doubt, minor release" - and I think this qualifies, as `from_type` will be much more powerful for protocol classes now.  (and so "patch" -> "release" below)

Reply:
cool cheers for clearing that up.